Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -283,24 +283,25 @@ function vtkWidgetManager(publicAPI, model) {
     // Default cursor behavior
     model._apiSpecificRenderWindow.setCursor(widget ? 'pointer' : 'default');
 
+    let wantRender = false;
     if (model.widgetInFocus === widget && widget.hasFocus()) {
       activateHandle(widget);
-      // Ken FIXME
-      model._interactor.render();
-      model._interactor.render();
+      wantRender = true;
```

Comment:
it is useless to be set to true, `wantRender` is not used afterward

Reply:
Whoops, I didn't pay attention to my indents. The `if (wantRender)` should be outside the if/else.

Reply:
Fixed.